corehq/apps/reports/standard/deployments.py

```diff
@@ -165,18 +165,20 @@ def rows(self):
 
         user_id = self.request.GET.get('individual')
         if not user_id:
-            return []
-
-        # security check
-        get_document_or_404(CommCareUser, self.domain, user_id)
-
+            key_args = {}
```

Comment:
Ah ha. Yeah, this definitely won't work. I don't see another view for `SyncLog`s. Easiest thing I think would be to update the placeholder text to say "Select a user" or something like that.